src/api/Gemfile.lock

```diff
@@ -116,11 +116,11 @@ GEM
     faker (1.7.3)
       i18n (~> 0.5)
     feature (1.4.0)
+    ffi (1.9.17)
```

Comment:
The same thing happens for me when I run bundle install, for some reason the ffi gem in Gemfile.lock is placed in the wrong line (alphabetically). This PR fixes that so IMO we should keep this change.